pkg/util/storage.go

```diff
@@ -0,0 +1,36 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Could we create a dedicated package for this (`persistentcache` or something). I think `util.StoreValue` is not very explicit.

Reply:
You mean something in `pkg` directly?